portal/templates/CourseCreation.html

```diff
@@ -0,0 +1,25 @@
++{% extends "layouts/base.html" %}
```

Comment:
this file was renamed 'course-creation.html' and does not need to be added back in with the old name